### ISSUE TITLE
bug in Recorder.read()

### ISSUE THREAD

**Issue by sarnoult** [annotation/text-fabric#69]
It looks like there is a bug in the Recorder.read() method. Right now it reads the text file, but writes to the `pos` file:
```
with open(posPath, "w", encoding="utf8") as fh:
        self.nodesByPos = [
            {int(n) for n in line.rstrip("\n").split("\t")} for line in fh
        ]
```

**Comment by dirkroorda** [comment]
Sure!
Sorry about that.
I'll fix it asap.
Thanks for noticing it, @sarnoult

**Comment by dirkroorda** [comment]
The fix is in version 8.4.12, which I just released.

```
pip3 install --upgrade text-fabric
```
